pycroquet/cli.py

```diff
@@ -291,7 +292,7 @@ def dual_guide(*args, **kwargs):
 @debug_params
 def long_read(*args, **kwargs):
     """
-    log-read - map read file to library guides and output counts, statistics and alignments files.
+    Map read file to library guides and output counts, statistics and alignments files.
```

Comment:
Do we need to take into account the library-independent quantification that was built in with the `unique_only` option for the description? i.e. we don't always map to the library? @keiranmraine

Reply:
Hmm, how to describe as part of that is a little tricky.  I wonder if we should just create a separate sub-command, thoughts (and idea what to call it).

Reply:
How about (if not adding an additional sub-command):

> Map read file to library guides and output counts, statistics and alignments files.  Minimal assessment of unique reads only via `--unique` option.